### DISCUSSION TITLE
libreelec and pivpn request support

### DISCUSSION THREAD

**Discussion by rijkie** [pivpn/pivpn#1395] [Feature requests]
## Issue
First I will tell, what I want to accomplish.
I want to [stream Xbox One to Windows 10 from ANYWHERE with Internet](https://kinkeadtech.com/how-to-stream-xbox-one-to-windows-10-from-anywhere-with-internet/). Since there is a [security issue when you need to open some port](https://kinkeadtech.com/question/great-article-but-security/). The best way for now is to create a vpn server. 
That's why I want to create a vpn server on my raspberry pi with libreelec.  I came across pivpn. But I am using libreelec not raspian. 

Could pivpn work on libreelec?
I was thinking that pivpn need a webserver/webui to be able to change some settings of pivpn through a a webbrowser? 
Than I can ask [thoradia](https://github.com/thoradia/thoradia) on github to make an addon of pivpn to let it work on libreelec.

## Have you searched for similar issues and solutions? 
No

**Comment by orazioedoardo** [comment]
LibreELEC is based on Kodi, so you could just install Kodi on a standard Raspbian image, then go for pivpn.

**Comment by rijkie** [comment]
I use libreelec because it is very easy to install some software apss that are available through addons on kodi. [https://github.com/thoradia/thoradia](https://github.com/thoradia/thoradia) - [https://github.com/linuxserver](https://github.com/linuxserver)

I don't know how to install these kind of software on raspian. Is that just as easy? Preferable without any keyboard input (only with  mouse or remote).  My understanding is that it is not that easy. That is why requested webserver/webui support. Although my knowledge how all these things work is limited.

**Comment by orazioedoardo** [comment]
I don’t know whether installing those addons on raspbian is easy or not, you should ask on their repository.

Installing pivpn on raspbian is very easy since is mostly automatic with a pseudo-gui, hence you shouldn’t encounter problems. To install kodi you just need `sudo apt install kodi` I think. You need to either hook up the raspberry to a monitor + keyboard + mouse or connect via ssh.

It could also be possible to install OpenVPN (in server mode) on LibreELEC itself but since it is a stripped down linux, may be hard to accomplish.

**Comment by coolapso** [comment]
That might bee a cool improvement .. but I will be honest .. I am putting this as not a priority since our resources are WAAAAY To limited!!! If anyone wants to roll up the sleeves go on!!

**Comment by alonpeer** [comment]
I was also exploring how to get this done. I was just able to use https://hub.docker.com/r/giggio/openvpn-arm/ as an alternative, and it seems to be working well.
